apps/src/templates/Button.jsx

```diff
@@ -6,11 +6,11 @@
 import _ from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
-import Radium from 'radium'; // eslint-disable-line no-restricted-imports
-import color from '@cdo/apps/util/color';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import classNames from 'classnames';
+import moduleStyles from './button.module.scss';
```

Comment:
Button and DropdownButton were converted from old `styles` constants and Radium, to new SCSS Modules.